README.md

```diff
@@ -35,8 +42,34 @@ school.roster
 # => {9 => ["Zach Morris"]}
 ``` 
 
-You can, of course, add several students to the same grade, and students to 
-different grades.
+**Hint:** If the roster hash starts off as being empty, how will you add key/value pairs to it, where the value is an array? Let's take a look at an example: 
+
+```ruby
+hash = {}
+```
+
+We start off with an empty hash above. Our desired result is a hash with a key that points to a value of an array. That array should contain a series of items. What happens if we try to create the key/value pair we want *and* add an item to the value array, all at the same time?
+
+```ruby
+hash["new_key"] << "new_value_for_value_array"
+ => NoMethodError: undefined method `<<' for nil:NilClass
+```
+
+We get an error! We can't push an item into an array that is the value of a key that doesn't exit yet! So, we *first* need to create the new key and point it to an empty array. *Then* we can push the new value into that array. 
+
+```ruby
+hash[new_key] = []
+hash[new_key] << new_value_for_value_array
+
+hash
+ => {"new_key"=>["new_value_for_value_array"]} 
+```
+
+Ta-da! When we first create the key and point it to a value of an empty array, we are able to then successfully add items to that array. 
+
+**Hint:** Now that we've thought about how to build up our hash, let's talk about how to add successive students to each grade. Let's say that we're adding the student `"AC Slater"` to grade `9`. If the `roster` already has a key of grade `9`, we'll be able to push AC Slater into the array that the grade `9` key points to. Otherwise, we'll need to first create the key of grade `9` and point it to an empty array. However, if we're in the scenario in which the grade `9` key already exists, we will erase it's current contents by creating a key of `9` and setting it equal to an empty array! You'll need to understand this in order to get the tests passing. 
```

Comment:
if you understood the hashes readings/labs inside and out this shouldn't be a problem for you but I ID'd this as a pain point for normal/average students, especially if you are doing this at home by yourself. it's in here as a reminder/hint/bit of hand-holding.